bundles/org.openhab.binding.airq/README.md

```diff
@@ -0,0 +1,222 @@
+# air-Q Binding
+
+The air-Q Binding integrates the air analyzer <a href="http://www.air-q.com">air-Q</a> device into the openHAB system.
+
+With the binding, it is possible to subscribe to all data delivered by the air-Q device.
+
+<img src="src/main/resources/image_air-Q.png" alt="air-Q image" width="400px" height="324px" />
```

Comment:
Readme assets should be located in the `doc/` folder in the root of this binding.